docs/pages/docs.md

```diff
@@ -5,20 +5,58 @@ layout: default
 
 # Documentation
 
-## Doc building
-
-| Name                                            | Short description                                               | 🚦  |
-| ----------------------------------------------- | --------------------------------------------------------------- | :-: |
-| [sphinx](https://www.sphinx-doc.org/en/master/) | Generates documentation from reStructuredText or Markdown files | 🟢  |
-| [gitbook](https://www.gitbook.com/)             | General documentation builder; integrates with GitHub           | 🟠  |
-| [mkdocs](https://www.mkdocs.org/)               | Generates documentation from Markdown files                     | 🟠  |
-| [pdoc](https://pdoc.dev/)                       | Auto-generates API documentation from docstrings                | 🟠  |
-
-<details>
-<summary> 🟢 explanation</summary>
-Sphinx is the de-facto standard that is widely used. It is well tested, reliable and very customisable.
+With Python, as with many other languages, it's very common to automatically
+create a web page with documentation built from [docstrings] in your package's
+API. For your package to be easily reusable we recommend you build documentation
```

Comment:
```suggestion
_application programming interface_ (API). For your package to be easily reusable we recommend you build documentation
```
Possibly overkill but even for commonly used acronyms I think its worth defining in full on first usage though we didn't define API in its usage before these edits so feel free to ignore